templates/lune/partials/components/chapter-name.snip.html

```diff
@@ -0,0 +1,3 @@
+{{#chapter}}
```

Comment:
do  you reallly need this to be a partial is this being reused ? i am sure you can group them into one partial if needed?